test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -9,13 +9,15 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
         if ($IsWindows) {
             $userName = "testuserservices"
             $testPass = "Secret123!"
```

Comment:
@TravisEz13 Can we use a clear text password?

Reply:
Even in tests, we should generate a password.

Reply:
Please generate a password

There is a function to generate a password here: 
https://github.com/PowerShell/PowerShell/blob/6118c37ce378cb1e092b1773568848ad79f67e09/test/powershell/Modules/Microsoft.PowerShell.Security/certificateCommon.psm1#L71

Reply:
@TravisEz13  will this function be available in the context or do we need to explicitly import certificateCommon.psm1 ?

Reply:
@kvprasoon You could insert
Import-Module (Join-Path -Path $PSScriptRoot 'certificateCommon.psm1') -Force